src/github/queries.js

```diff
@@ -17,9 +17,73 @@ const me = () => {
   });
 };
 
+const simpleContributorFields = `
+  id
+  name
+  bio
+  location
+  email
+  login
+  websiteUrl
+  avatarUrl
+`;
+
+const contributor = (login) => {
+  const data = simpleContributorFields;
+  const variables = {login};
+  const inputs = '';
+  return getContributorData(inputs, data, variables);
+}
+
+const isContributor = (login, orgID) => {
+  const inputs = '$orgID: ID!';
+  const data = `
+    contributionsCollection(organizationID: $orgID) {
+      hasAnyContributions
+    }
+  `;
+  const variables = { login, orgID };
+  return getContributorData(inputs, data, variables);
```

Comment:
why you need this?

Reply:
Validation about login parameter on contributor(login) query